ui/src/main/resources/Meeting/Code/Translations.xml

```diff
@@ -97,6 +97,15 @@ meeting.extension.name=Meeting Application (Pro)
 meeting.durationDisplay.label=Meeting Duration
 meeting.home.migrations=Some properties have been changed since your last upgrade. Please follow the next link/s to complete the migration:
 
```

Comment:
You're missing translation for the administration section title and description. See https://github.com/xwikisas/application-licensing/blob/master/application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/Translations.xml#L39-L40 .